doc/API.md

```diff
@@ -78,7 +78,14 @@ The currently-defined error responses are:
 
 * Setup
     * [POST /setup](#post-setup)
-    * [POST /login](#post-login)
+* Login
+    * [POST /login](#post-login) :lock:
+* User management
+    * [POST /](#post-) :lock:
```

Comment:
Oh!
This middleware is going to be mounted in `/users/` already, ok ok, forget my comment :)